app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -623,15 +623,13 @@ export const ContentForm: React.FC<IContentFormProps> = ({
                                 </Show>
                               </Row>
                             </Tab>
-                            <Tab
```

Comment:
Please just comment out both tabs.  They are still needed, but not presently.

Reply:
Updated, thanks